src/components/Dashboard/Dashboard.jsx

```diff
@@ -1,11 +1,36 @@
 import React from 'react';
 
 import Header from 'components/Header/Header';
+import IngredientCard from 'components/IngredientCard/IngredientCard';
 import SearchBox from 'components/SearchBox/SearchBox';
+import Themes from 'assets/json/themes.json';
+import Chip from 'components/Chip/Chip';
 
+import Milk from 'images/ingredients/milk.png';
+import Yogurt from 'images/ingredients/yogurt.png';
+
+// TODO: Move all the place components in their own area
 const Dashboard = () => (
   <div className="dashboard">
     <Header />
+    <IngredientCard
+      name="Milk"
+      image={Milk}
+      theme={Themes.teal}
+    >
+      <Chip name="calcium" />
```

Comment:
You should create template json files for this! You should run a map over the array in the json and get these so you dont have to write each one separately

Reply:
These are properties on the Ingredient model and will come from the backend. This is just a prototype display. The dashboard component will not be like that. In the development stage for components it is just a display.